From: Pietro Battiston <me@pietrobattiston.it>
Date: Wed, 21 Dec 2011 15:29:24 +0100
Subject: disable-broken-locale-test

The following test assumes that the locale "pt_BR.UTF-8" is
installed, and fails otherwise. Since there is no "non-trivial" locale which we
can assume is installed, the only way to fix this seems to disable it.
Forwarded, but upstream seems willing to not drop the test.
Author: Pietro Battiston <me@pietrobattiston.it>
Forwarded: yes
---
 shapely/tests/wkt_locale.txt | 28 ----------------------------
 1 file changed, 28 deletions(-)
 delete mode 100644 shapely/tests/wkt_locale.txt

diff --git a/shapely/tests/wkt_locale.txt b/shapely/tests/wkt_locale.txt
deleted file mode 100644
index ebbc024..0000000
--- a/shapely/tests/wkt_locale.txt
+++ /dev/null
@@ -1,28 +0,0 @@
-Test locale independence of WKT
-===============================
-
-Set locale to one that uses a comma as decimal seperator`
-
-  >>> import locale, sys
-  >>> if sys.platform == 'win32':
-  ...     _ = locale.setlocale(locale.LC_ALL, 'portuguese_brazil')
-  ... else:
-  ...     _ = locale.setlocale(locale.LC_ALL, 'pt_BR.UTF-8')
-
-Test reading and writing`
-
-  >>> from shapely.wkt import loads
-  >>> p = loads('POINT (0.0 0.0)')
-  >>> p.x == 0.0
-  True
-  >>> p.y == 0.0
-  True
-  >>> p.wkt
-  'POINT (0.0...)'
-  >>> from shapely.wkt import dumps
-  >>> dumps(p)
-  'POINT (0.0...)'
-
-Reset locale
-
-  >>> _ = locale.setlocale(locale.LC_ALL, '')
